web/packages/teleterm/package.json

```diff
@@ -46,7 +44,7 @@
     "@types/tar-fs": "^2.0.1",
     "@types/whatwg-url": "^11.0.1",
     "clean-webpack-plugin": "4.0.0",
-    "electron": "28.2.3",
+    "electron": "29.1.1",
```

Comment:
Do you mind updating it to 29.1.4? I see that [there's a couple of fixes for crashes on Windows](https://releases.electronjs.org/release/compare/v29.1.1/v29.1.4), I think it'd be worthwhile to have them in. Just running a tag build + checking if it runs on Windows should be enough wrt testing.

Reply:
Done, I tested it on 15.0.0-dev.gzdunek.13.